### PR TITLE
Add Responsive

### PR DESCRIPTION
**Pull request by adrianncovaci**

### CODE REVIEW

File: src/pages/MainPage/MainPage.styled.ts [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import styled, { DefaultTheme, StyledComponent } from 'styled-components';
-import BannerImage from '../../assets/resources/banner.png';
-
 
 export const Container: StyledComponent<'div', DefaultTheme> = styled.div`
   font-family: 'DM Serif Display';
-  max-width: 375px;
   min-width: 375px;
+
```
Comment (cristian-delinschi) [inline src/pages/MainPage/MainPage.styled.ts]: woaw

File: src/pages/MainPage/MainPage.styled.ts [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
+  @media only screend and (min-width:768px) {
```
Comment (cristian-delinschi) [inline src/pages/MainPage/MainPage.styled.ts]: nifiga se

File: src/pages/MainPage/MainPage.styled.ts [hunk 3]
```diff
@@ -1,9 +1,10 @@
...
+    width: 800px;
```
Comment (cristian-delinschi) [inline src/pages/MainPage/MainPage.styled.ts]: asta-i moshi

Comment (strdr4605) [reply src/pages/MainPage/MainPage.styled.ts]: ❤️

**Pull request merged**